### DISCUSSION TITLE
v2.11.2

### DISCUSSION THREAD

**Discussion by mienaiyami** [mienaiyami/yomikiru#144] [General]
## Downloads

[![Download setup](https://img.shields.io/badge/Windows%20Setup%20(exe)-Yomikiru--2.11.2--Setup.exe-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.11.2/Yomikiru-2.11.2-Setup.exe)

[![Download Portable](https://img.shields.io/badge/Windows%20Portable%20(zip)-Yomikiru--win32--2.11.2--Portable.zip-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.11.2/Yomikiru-win32-2.11.2-Portable.zip)

---
## Note

Linux distribution currently not available as direct download but can be build easily by following this [Build for linux(debian) guide](https://github.com/mienaiyami/yomikiru/blob/master/docs/build_for_linux.md).

---

## Added

- Tab (keyboard) navigation for "reader settings's tabs".
- New scrolling system (with keys).
- Custom stylesheet.
- New shortcut: `ctrl+w` to close window.
- EPUB : Force low brightness.

## Changed

- UI Enhancements.
- Save data files with a time gap after changing app data.

## Fixed

- fixed #131,#136,#137,

---

#### Todo

- Write tests.
- Add ability to change image quality in canvas mode for less RAM usage and performance.

---

**Full Changelog**: https://github.com/mienaiyami/yomikiru/compare/v2.11.1...v2.11.2

<hr /><em>This discussion was created from the release <a href='https://github.com/mienaiyami/yomikiru/releases/tag/v2.11.2'>v2.11.2</a>.</em>

**Comment by zeoint** [comment]
What is the scroll speed px / scroll size px of mouse navigation used in book reader and epub reader?

**Reply by mienaiyami** [reply]
It is not handled by the app. It is set by windows (check your mouse settings in windows settings).

**Reply by zeoint** [reply]
Great.
